docs/csharp/how-to/index.md

```diff
@@ -0,0 +1,241 @@
+- [Use the xml documentation features to document your code](../programming-guide/xmldoc/how-to-use-the-xml-documentation-features.md).
+- [Explicitly implement interface members](../programming-guide/interfaces/how-to-explicitly-implement-interface-members.md) to keep your public interface more clear.
+- [Explicitly implement members of two interfaces](../programming-guide/interfaces/how-to-explicitly-implement-members-of-two-interfaces.md).
+
+### Working with collections
+
+These articles help you work with collections of data.
+
+- [Initialize a dictionary with a collection initializer](../programming-guide/classes-and-structs/how-to-initialize-a-dictionary-with-a-collection-initializer.md).
+- [Access all the elements in a collection using `foreach`](../programming-guide/classes-and-structs/how-to-access-a-collection-class-with-foreach.md).
+
+## strings
+
+Strings are the fundamental data type used to display or manipulate text. These articles demonstrate common practices with strings.
+
+- [Compare strings](../programming-guide/strings/how-to-compare-strings.md).
+- [Modify the contents of a string](../programming-guide/strings/how-to-modify-string-contents.md).
+- [Determine if a string represents a number](../programming-guide/strings/how-to-determine-whether-a-string-represents-a-numeric-value.md).
+- [Use <xref:System.String.Split%2A> to separate strings](../programming-guide/strings/how-to-parse-strings-using-string-split.md).
+- [Combine multiple strings into one](../programming-guide/strings/how-to-concatenate-multiple-strings.md).
+- [Search for text in a string](../programming-guide/strings/how-to-search-strings-using-string-methods.md).
+- [Search strings using regular expressions](../programming-guide/strings/how-to-search-strings-using-regular-expressions.md).
+
+## Convert between types
+
+You may need to convert an object to a different type.
+
+- [Determine if a string represents a number](../programming-guide/strings/how-to-determine-whether-a-string-represents-a-numeric-value.md).
+- [Convert between strings that represent hexadecimal numbers and the number](../programming-guide/types/how-to-convert-between-hexadecimal-strings-and-numeric-types.md).
+- [Convert a string to a <xref:System.DateTime>](../programming-guide/strings/how-to-convert-a-string-to-a-datetime.md).
+- [Convert a byte array to an int](../programming-guide/types/how-to-convert-a-byte-array-to-an-int.md).
+- [Convert a string to a number](../programming-guide/types/how-to-convert-a-string-to-a-number.md).
+- [Use `as` and `is` to safely cast to a different type](../programming-guide/types/how-to-safely-cast-by-using-as-and-is-operators.md).
+- [Define conversion operators for `struct` types](../programming-guide/statements-expressions-operators/how-to-implement-user-defined-conversions-between-structs.md).
+- [Determine if a type is a nullable type](../programming-guide/nullable-types/how-to-identify-a-nullable-type.md).
```

Comment:
or just "Determine if a type is nullable"?

Reply:
I changed to nullable value type.  I want to future-proof this against the upcoming non-nullable reference types. (The referenced article doesn't help determine nullable vs. non-nullable reference types, only value types.)